crates/turbopack-core/src/resolve/mod.rs

```diff
@@ -341,7 +341,6 @@ impl ModuleResolveResultOption {
 #[derive(Clone, Debug)]
 pub enum ResolveResultItem {
     Source(Vc<Box<dyn Source>>),
-    OriginalReferenceExternal,
```

Comment:
This has been removed in favor of only `OriginalReferenceTypeExternal`

Reply:
finally haha

but maybe rename `OriginalReferenceTypeExternal` to `ExternalReference` 🤔